### DISCUSSION TITLE
Scroll anywhere to change volume when eartrumpet is open

### DISCUSSION THREAD

**Discussion by KLuuKer** [File-New-Project/EarTrumpet#658] [Ideas]
I want to be able to just open the menu and scroll everywhere else to change the volume
Just like with the stock windows volume control

![volume-scroll](https://user-images.githubusercontent.com/12729178/41371921-1951ef64-6f4c-11e8-854e-bf2bc01f5ff2.gif)

**Comment by do-bo** [comment]
I found interesting behaviour/bug related to this - scrolling anywhere changes volume but it also scrolls the window under the cursor.

To reproduce (v 2.0): Have at least 2 playback devices. Unfold the devices flyout (if only the default device is shown). Fold the full list back so that only the default device is shown again. Now you can scroll the volume with mouse cursor anywhere, but anything under the cursor is also being scrolled.

**Comment by riverar** [comment]
Yep got it, thanks!

**Comment by Epicoss** [comment]
Or if that isn't an option make it that you can scroll to change the volume when the cursor is on the start bar intead of on the ear trumpet icon. It isn't that practical to have such a small area.

**Comment by b82a** [comment]
Very much needed

**Comment by Tester798** [comment]
@b82a you can try my implementation of this (and few other) function at https://github.com/Tester798/EarTrumpet/tree/dev-custom

**Comment by vertigo220** [comment]
As long as this is optional, because I *hate* that "feature" of Win10. I'm constantly changing the volume accidentally while trying to scroll something else, since this behavior is completely contrary to the standard. In fact, they _finally_ added the ability to scroll background windows in Win10, something I had to use a 3rd-party app for in 7, and then they implement this behavior, which doesn't follow that option's setting. Typical M$...

Anyways, I'd suggest you check out 7++ Taskbar Tweaker, which has the option to adjust the volume when scrolling over the notification area or the entire taskbar, as well as many other very useful features that should be built in to Windows.

**Comment by saivajr** [comment]
Is there a way to make an setting for that? I saw a similar request about it that was supposedly added on v2.0 but I'm on 2.1.10.0 and there seems to be no way of doing that at the moment. It's the only thing keeping me from removing the default volume icon from my taskbar!

**Comment by Tester798** [comment]
Created PR for this: #1168, any comments are welcome.

**Comment by Wolverine1977** [comment]
Can you add the ability to change the volume by scrolling anywhere on the taskbar - mute /unmute with middle click anywhere on taskbar

**Reply by vertigo220** [reply]
7+ Taskbar Tweaker can do this and a lot more. Not saying it shouldn't be considered for this project, but you can do it now without waiting for that.

**Reply by Wolverine1977** [reply]
"7+ Taskbar Tweaker doesn’t support the Windows 11 taskbar, and probably never will."
Also tried TB-Vol-Scroll but it doesnt do Mute on middle click and looks ugly in comparison to Volume2
